src/main.rs

```diff
@@ -383,13 +383,34 @@ impl fmt::Display for UnixSocketEntry {
             None => self.socket_type.clone(),
         };
         if let Some(peer) = &self.peer {
-            let m = format!(" -> {}[{}]", peer.endpoint.name, peer.endpoint.pid);
-            msg.push_str(&m);
+            // Group endpoints by name and pid
+            // Eg: foo[123][4],foo[123][5] => foo[123][4,5]
```

Comment:
I'm not sure I completely grok this comment. Is this what we expect the output to end up looking like?

Reply:
Yes, instead of displaying `foo[123][4],foo[123][5]` , it will group fd's to show just `foo[123][4,5]`